java/java.mx.project/src/org/netbeans/modules/java/mx/project/Compliance.java

```diff
@@ -99,12 +99,23 @@ public String toString() {
         }
     }
 
+    boolean matches(Compliance check) {
+        if (min > check.min) {
```

Comment:
`>` or `>=` ?

Reply:
`return false` if outside of range, but you are right, there should be a test: 1ce26972e9f0